src/native/corehost/apphost/static/CMakeLists.txt

```diff
@@ -42,6 +42,20 @@ include(../../fxr/files.cmake)
 include(../../hostpolicy/files.cmake)
 include(../../hostcommon/files.cmake)
 
+if(MSVC)
+    add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/wd4996>)
```

Comment:
these `wd` defined when everything in host.native is built. 
Since we build this separately from others now, I had to add the options here.